packages/xchain-litecoin/src/client.ts

```diff
@@ -33,7 +33,8 @@ interface LitecoinClient {
 type LitecoinClientParams = XChainClientParams & {
   sochainUrl?: string
   nodeUrl?: string
-  nodeAuth?: NodeAuth
+  // Pass strict null to disable auth for node json rpc
```

Comment:
Could you put this important information into comment of constructor, pls? It will be visible for auto-generated documentation then.